tests/flow/test_access_del_entity.py

```diff
@@ -191,20 +190,12 @@ def test08_create_edge_to_deleted_node(self):
     def test09_path_with_deleted_node(self):
         # test path with deleted node
         # create a 3 nodes path (a)->(b)->(c)
-        a = Node(label="A", properties = {'v':'a'})
-        b = Node(label="B", properties = {'v':'b'})
-        c = Node(label="C", properties = {'v':'c'})
-        self.g.add_node(a)
-        self.g.add_node(b)
-        self.g.add_node(c)
-
-        r1 = Edge(a, 'R', b)
-        r2 = Edge(b, 'R', c)
-
-        self.g.add_edge(r1)
-        self.g.add_edge(r2)
-
-        self.g.flush()
+        a  = Node(alias="a", labels="A", properties = {'v':'a'})
+        b  = Node(alias="b", labels="B", properties = {'v':'b'})
+        c  = Node(alias="c", labels="C", properties = {'v':'c'})
+        r1 = Edge(a, 'R', b, alias="r1",)
+        r2 = Edge(b, 'R', c, alias="r2")
+        self.g.query(f"CREATE {a}, {b}, {c}, {r1}, {r2}")
```

Comment:
I don't want to allow users to create graphs with `Node` and `Edge` objects. (Indeed this might had been convenient in our test suite)
But this was one of the big changes we've made in the Python API.